charts/trino/templates/configmap-coordinator.yaml

```diff
@@ -160,6 +166,21 @@ data:
 
 ---
 
+{{- if .Values.sessionPropertyConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: trino-session-property-config-volume-coordinator
```

Comment:
I'm not sure historically as to why some config is deployed to a new configmap while everything else (node.properties, jvm.config etc) is deployed into a single configmap?

However if a separate configmap is still desired, please avoid hardcoding the name and instead make use of a template function? See the naming used for the main configmap and the example in #109 .

It is helpful (for testing etc) to be able to ensure that multiple trino instances can be deployed into a single namespace.